testcases/crypto/ec_func.c

```diff
@@ -2392,13 +2388,15 @@ int main(int argc, char **argv)
     rv += run_DeriveECDHKey();
     rv += run_DeriveECDHKeyKAT();
 
-    pkey = CK_TRUE;
-    rv = run_GenerateECCKeyPairSignVerify();
-    rv += run_ImportECCKeyPairSignVerify();
-    rv += run_TransferECCKeyPairSignVerify();
-    rv += run_DeriveECDHKey();
-    rv += run_DeriveECDHKeyKAT();
-    rv += run_ImportSignVerify_Pkey();
+    if (is_ep11_token(SLOT_ID)) {
+        pkey = CK_TRUE;
```

Comment:
Yes, that make sense. But if we check is_ep11_token() here, we can remove the is_ep11_token() check inside the various routines.

Reply:
yea, I thought about this, but I guess it does not hurt to keep the check there as well. Just in case someone calls these functions with pkey set to true.